SERVICE_ACCOUNT.md

```diff
@@ -0,0 +1,33 @@
+This file gives guidelines for organization how they can create and configure service account
```

Comment:
Please put all this in the `README.md` in a section `## Releasing in your organisation`

Reply:
@kelson42 Done.